### ISSUE TITLE
[Suggestion] Use Chum only when GP is greater than certain amount&Use ThaliaksFavor after Identical Cast for double/triplehook

### ISSUE THREAD

**Issue by Keinechanw** [InitialDet/AutoHook#27]
Would you consider add a GP detection before Chum so that it can save enough GP for other ablities?

Also, maybe using ThaliaksFavor after Identical Cast for double/triplehook would be a good idea.

Both of them should be easy to implement by adding one line:)

**Comment by InitialDet** [comment]
Sure, i'll see what i can do

**Comment by InitialDet** [comment]
I've tried making action combos before but they were a pain in the ass because there are lots of conditions and priorities for the auto casts, and managing the GP was a nightmare.

In this case they are quite simple I suppose, but it might take me some time.

**Comment by Keinechanw** [comment]
> I've tried making action combos before but they were a pain in the ass because there are lots of conditions and priorities for the auto casts, and managing the GP was a nightmare.
> 
> In this case they are quite simple I suppose, but it might take me some time.

Indeed. Managing all condition would be impossible, but managing this one won't have to consider much.

**Comment by InitialDet** [comment]
Sorry for the delay. I've added the GP validation for now. I'll look into the combo feature in a later version to avoid conflicts

**Comment by SiliconExarch** [comment]
Would it also be possible to add Identical Cast to the GP Config tab since I'd like to be able to restrict its use to when I have almost enought GP to pop off a double/triple hook directly afterwards rather than any time I have more than the 350 required for the action itself.
